src/edu/ucsd/library/xdre/tab/FilesChecker.java

```diff
@@ -154,6 +172,10 @@ public boolean filesValidate ()
 			// Build the command for file validation
 			List<String> cmd = new ArrayList<>();
 			cmd.add( command );
+			if (src.length() > MAX_FILESIZE) {
```

Comment:
I don't think it's an issue.  I just took a look at the Perl code (from http://www.sno.phy.queensu.ca/~phil/exiftool/) and it looks like it's just allowing byte offsets larger than 2GB, not allocating extra buffers or anything like that.  So I think it's fine to enable it all the time.